pkgs/games/vvvvvv/default.nix

```
@@ -80,8 +71,13 @@ stdenv.mkDerivation rec {
 
     install -Dm755 VVVVVV $out/bin/${pname}
     install -Dm644 "$src/desktop_version/icon.ico" "$out/share/pixmaps/VVVVVV.png"
+    cp -r "$src/desktop_version/fonts/" "$out/share/"
+    cp -r "$src/desktop_version/lang/" "$out/share/"
 
-    wrapProgram $out/bin/${pname} --add-flags "-assets ${dataZip}"
+    wrapProgram $out/bin/${pname} \
```

```suggestion
    wrapProgram $out/bin/vvvvvv \
```
this shouldn't rely on pname